http/stats.md

```diff
@@ -0,0 +1,54 @@
+| started            | sum                      | Number of all client requests started                                                                                                                                                                                                                                                             |
+| bytes_sent         | sum                      | Total bytes sent in request body (not including headers). This is uncompressed bytes.                                                                                                                                                                                                             |
+| bytes_recv         | sum                      | Total bytes received in response bodies (not including headers but including error responses with bodies). Should be measured from actual bytes received and read, not the value of the Content-Length header. This is uncompressed bytes. Responses with no body should record 0 for this value. |
+| headers_sent       | none                     | Total number of header lines sent in outgoing requests, not including trailing headers                                                                                                                                                                                                            |
+| headers_recv       | none                     | Total number of header lines received in responses, not including trailing headers                                                                                                                                                                                                                |
+| latency            | distribution             | Time between first byte of request headers sent to last byte of response received, or terminal error                                                                                                                                                                                              |
+| ttfb               | distribution             | Time between last byte of request sent and first byte of response body received                                                                                                                                                                                                                   |
+| connections_opened | count                    | Number of underlying transport (TCP/TLS) connections opened                                                                                                                                                                                                                                       |
```

Comment:
Is this the total number of connection or is the active connections?

Reply:
I imagined it was just the total number opened. I can see the latter being useful as well so I'll add a connections_closed as well.

Reply:
Sounds good.